csm/core/blogic/const.py

```diff
@@ -970,3 +970,7 @@
 SWAGGER_JSON_URL = '/api/doc/swagger.json'
 SWAGGER_UI_STATICS_URL = '/api/static'
 
+# Version Validation
+REQUIRES = 'requires'
+INFORMATION_SERVICE = "information_service"
+VERSION_RESOURCES = ['cluster', 'node']
```

Comment:
Is cluster as a resource is allowed from provisioner ?

Reply:
not yet.. as it was mentioned in REST API specification for supported resource.. Thats why I kept it for now